.jenkins/pytorch/build-mobile.sh

```diff
@@ -26,11 +26,6 @@ retry pip install --pre torch torchvision \
 # binary, and running forward pass with a real model.
 if [[ "$BUILD_ENVIRONMENT" == *-mobile-custom-build-static* ]]; then
   TEST_CUSTOM_BUILD_STATIC=1 test/mobile/custom_build/build.sh
```

Comment:
Maybe we could change the name later. "TEST_CUSTOM_BUILD_STATIC" looks confusing that we are still doing static selective build? cc @cccclai

Reply:
Oh, it may be related to selective build with static dispatch.

Reply:
Yes, it's selective build with static dispatch. The static analyzer one is below, called "mobile-custom-build-dynamic", as it's created for dynamic dispatch.